src/shortcuts/note.ts

```diff
@@ -69,7 +69,7 @@ const noteShortcut: Shortcut = {
     const state = getState()
     const { cursor } = state
     const context = pathToContext(cursor ? simplifyPath(state, cursor) : HOME_PATH)
-    return attribute(state, context, '=note') != null
+    return attribute(state, context, '=note') != null || isShortcutExecutable(getState)
```

Comment:
This would destroy the meaning of the existing `isActive`, as `isShortcutExecutable` will always return true when the cursor is valid, regardless of `attribute`.

We should either use the existing behavior (dim when there is no note) or the new functionality (dim when it is not possible to add a note). I would keep the existing behavior.

Some shortcuts will dim when they are not executable, and some shortcuts will dim when their respective attribute is disabled on the cursor. This is currently ambiguous to the user, which is a tad confusing. Yet it seems wrong to not dim a shortcut when it is not executable.

If that doesn't make sense that's okay. Just know that we should leave existing `isActive` methods as-is.

Reply:
Currently non-executable icon is also clickable. Should we look into making them non-clickable if they can not be executed?

Reply:
Could you explain what you mean by clickable? If `canExecute` returns false, clicking the toolbar button will do nothing. It will produce an animation though which I think should be disabled.

Reply:
What I mean is if action related to icon is not executable, user can still click on it and expect something to happen. Based on current set up, when user clicks on it, we will show them the animation, but nothing will happen which might be confusing for them. So, if toolbar icon is non-executable, we should disable it and stop the animation. Also on desktop, we should not have cursor as pointer. When cursor is changed to pointer, it means it is a link and user can click on it and expect something to happen.